numpy/add_newdocs.py

```diff
@@ -5874,7 +5882,20 @@ def luf(lamdaexpr, *args, **kwargs):
     >>> np.add.reduce(X, 2)
     array([[ 1,  5],
            [ 9, 13]])
-
+           
+    You can use the initializer argument to initialize the reduction to a
+    different value.
+    
+    >>> np.add.reduce([10], initializer=5)
+    15
+    >>> np.add.reduce(np.ones((2, 2, 2)), axis=(0, 2), initializer=10)
+    array([14., 14.])
+    
+    The initializer argument can be used to calculate empty reductions for
+    ufuncs that don't support it as well.
```

Comment:
Would maybe read better as "allows reductions of empty arrays where they would normally fail" - I don't like the `as well` at the end here.

Could also show how `np.minimum.reduce([])` fails